set.mm

```diff
@@ -128518,12 +128518,32 @@ Infinity and the extended real number system (cont.)
     ( clt wbr cle cxr wcel wa wceq wo orc xrleloe syl5ibr ) ABCDZABEDAFGBFGHNAB
     IZJNOKABLM $.
 
+  ${
+    xrltled.a $e |- ( ph -> A e. RR* ) $.
+    xrltled.b $e |- ( ph -> B e. RR* ) $.
+    xrltled.altb $e |- ( ph -> A < B ) $.
+    $( 'Less than' implies 'less than or equal to' for extended reals.
+       Deduction form of ~ xrltle .  (Contributed by Glauco Siliprandi,
+       11-Dec-2019.) $)
+    xrltled $p |- ( ph -> A <_ B ) $=
+      ( clt wbr cle cxr wcel wi xrltle syl2anc mpd ) ABCGHZBCIHZFABJKCJKPQLDEBC
+      MNO $.
+  $}
+
   $( 'Less than or equal to' is reflexive for extended reals.  (Contributed by
      NM, 7-Feb-2007.) $)
   xrleid $p |- ( A e. RR* -> A <_ A ) $=
     ( cxr wcel cle wbr wa clt wceq wo eqid olci xrleloe mpbiri anidms ) ABCZAAD
     EZOOFPAAGEZAAHZIRQAJKAALMN $.
 
+  ${
```

Comment:
The shortening with this theorem is a border case.  Each reference only saves a few bytes.  Even 36 occurrences do not add up to more than 2 saved lines of the file, which is more than the theorem itself needs.  Our rules forbid such 'shortenings'.

Reply:
Form the output of the minimization below, I count a 292 bytes reduction. So I'd like to keep it. That's not a theorem I contributed, I did this PR in order to take care of the list in #2128 and this is simply the PR #2778 which had to be closed because of merge conflicts, but had been approved.
```
Proof of "blcld" decreased from 2080 to 2068 bytes using "xrleidd".
Proof of "blss" decreased from 1425 to 1408 bytes using "xrleidd".
Proof of "blssps" decreased from 1438 to 1421 bytes using "xrleidd".
Proof of "deg1lt" decreased from 381 to 372 bytes using "xrleidd".
Proof of "difioo" decreased from 763 to 748 bytes using "xrleidd".
Proof of "eliccelico" decreased from 855 to 844 bytes using "xrleidd".
Proof of "elicoelioo" decreased from 914 to 903 bytes using "xrleidd".
Proof of "esumpmono" decreased from 994 to 989 bytes using "xrleidd".
Proof of "iccpartgel" decreased from 792 to 786 bytes using "xrleidd".
Proof of "iccpartleu" decreased from 753 to 747 bytes using "xrleidd".
Proof of "imasdsf1olem" decreased from 9511 to 9495 bytes using "xrleidd".
Proof of "infxrpnf" decreased from 1049 to 1043 bytes using "xrleidd".
Proof of "infxrre" decreased from 704 to 699 bytes using "xrleidd".
Proof of "iocinico" decreased from 837 to 828 bytes using "xrleidd".
Proof of "ismbl4" decreased from 544 to 538 bytes using "xrleidd".
Proof of "itg2eqa" decreased from 399 to 389 bytes using "xrleidd".
Proof of "iunhoiioo" decreased from 960 to 951 bytes using "xrleidd".
Proof of "limcresiooub" decreased from 2438 to 2430 bytes using "xrleidd".
Proof of "limsupbnd1" decreased from 645 to 640 bytes using "xrleidd".
Proof of "limsupbnd2" decreased from 1314 to 1307 bytes using "xrleidd".
Proof of "limsupgre" decreased from 3165 to 3158 bytes using "xrleidd".
Proof of "mdeglt" decreased from 619 to 613 bytes using "xrleidd".
Proof of "metds0" decreased from 703 to 699 bytes using "xrleidd".
Proof of "metdseq0" decreased from 1919 to 1913 bytes using "xrleidd".
Proof of "metdstri" decreased from 2238 to 2232 bytes using "xrleidd".
Proof of "nmolb" decreased from 590 to 584 bytes using "xrleidd".
Proof of "pcadd" decreased from 4043 to 4034 bytes using "xrleidd".
Proof of "pcdvdstr" decreased from 703 to 697 bytes using "xrleidd".
Proof of "sge0prle" decreased from 654 to 642 bytes using "xrleidd".
Proof of "signsply0" decreased from 4692 to 4684 bytes using "xrleidd".
Proof of "snunico" decreased from 403 to 399 bytes using "xrleidd".
Proof of "snunioo" decreased from 420 to 416 bytes using "xrleidd".
Proof of "supxrre" decreased from 754 to 750 bytes using "xrleidd".
Proof of "xadd0ge" decreased from 311 to 307 bytes using "xrleidd".
Proof of "xleadd1a" decreased from 1459 to 1444 bytes using "xrleidd".
Proof of "xrge0omnd" decreased from 679 to 666 bytes using "xrleidd".
```

Reply:
I would also vote for keeping the theorem.

Reply:
The theorem xrleidd itself occupies 268 bytes in set.mm.  So we save in total 24 bytes in all of set.mm, less if we look at Main only...  The shortening effect alone is not impressive in my eyes, but if you both see extra value in this theorem I won't further object.

Reply:
Not impressive indeed. This came in because I had to move a theorem from a mathbox (since there were two duplicates), and that theorem used xrleidd. So: not a regular thing.